helpers.py

```diff
@@ -3,38 +3,54 @@
 import pg8000
 import io
 
-def query_db(query, database='postgres', host='localhost', user='postgres', password='', response=True, autocommit=False):
-        psql_conn = pg8000.connect(host=host, database=database, user=user, password=password)
-        psql_conn.autocommit = autocommit
-
-
-        print('Querying DB')
+'''
+Collection of helper functions to use in the survey training preprocessor
+'''
 
-        cursor = psql_conn.cursor()
-        cursor.execute(query)
-
-        if response:
-            result = cursor.fetchall()
-            cursor.close()
-            psql_conn.close()
-            return result
-        else:
-            cursor.close()
-            psql_conn.close()
+def query_db(query, database='postgres', host='localhost', user='postgres', password='', response=True, autocommit=False):
+    '''
+    Function that connects to the specified PostgreSQL database and executes the given query
+    '''
+    psql_conn = pg8000.connect(host=host, database=database, user=user, password=password)
+    psql_conn.autocommit = autocommit
+
+    print('Querying DB')
+
+    cursor = psql_conn.cursor()
+    cursor.execute(query)
+
+    if response:
+        result = cursor.fetchall()
+        cursor.close()
+        psql_conn.close()
+        return result
+    else:
+        cursor.close()
+        psql_conn.close()
```

Comment:
possible change to singleton? Seems to weird to open and close connections every time :penguin:  should as an issue after merge